### ISSUE TITLE
A11y_AzureDataStudioSQLMigration_AzureSQLMigration_Zoom:The "Help articles and link" section is not clearly visible in 200% zoom.

### ISSUE THREAD

**Issue by Shivamd1** [microsoft/azuredatastudio#15736]
[“Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AZUREDATASTUDIOSQLMigration_June2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG1.4.4;#AINotInScope;#Zoom

### Pre-Requisite:
Zoom the page to 200%.

### Environment Details:
Application Name: SQL Azure Data Studio
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure SQL Migration.
2. Connect to any sql server.
3. Tab till Azure SQL migration Tab and press enter.
4. View the page in 200% zoom.

### Actual:
The "Help articles and link" section is not clearly visible in 200% zoom.

### Expected:
All the elements in the page should be clearly visible at 200% zoom.

### User Impact:
Users viewing the page at 200% zoom will not get the complete information.

**Comment by aasimkhan30** [comment]
This has to be fixed in the core as the ML extension dashboard also has this bug.

**Comment by Yashi1201** [comment]
Closing the bug as the issue is not repro
Version:1.42.0-insider(user-setup)
![15736_issue fixed](https://user-images.githubusercontent.com/93374043/218965909-0ef4ec15-ea2e-4b80-b2c2-0dffe83b8342.png)

**GithubTags**:#Closed;
